pages/common/git-summary.md

```diff
@@ -0,0 +1,17 @@
+# git summary
+
+> Display information about a Git repository.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-summary>.
+
+- Display data about a Git repository:
+
+`git summary`
+
+- Display data about a Git repository since a commit-ish:
+
+`git summary {{commit-ish}}`
```

Comment:
We user lowercase for placeholders, but I think you can split this example into 1 for a specific commit and another for a full branch, or maybe:
```suggestion
`git summary {{commit|branch_name|tag_name}}`
```